services/app/src/components/views/quest/cardtemplates/combat/Actions.test.tsx

```diff
@@ -239,6 +240,32 @@ describe('Combat actions', () => {
 
       expect(store.getActions()[1].node.ctx.templates.combat.levelUp).toEqual(false);
     });
+
+    it('Fixes when given a mid-combat roleplay node', () => {
```

Comment:
This test title feels unclear what it's actually testing?

Reply:
Done.